docs/src/SUMMARY.md

```diff
@@ -1,13 +1,15 @@
 # Table of contents
 
 * [Introduction](introduction.md)
-* [Install the Solana Tool Suite](install-solana.md)
-* [Command-line Guide](cli/README.md)
-  * [Choose a Wallet](cli/choose-a-wallet.md)
-  * [Hardware Wallets](remote-wallet/README.md)
-    * [Ledger Hardware Wallet](remote-wallet/ledger.md)
-  * [Paper Wallet](paper-wallet/README.md)
-    * [Paper Wallet Usage](paper-wallet/usage.md)
+* [Wallet Guide](wallet/README.md)
+  * [Web Wallets](wallet/web-wallets.md)
```

Comment:
Why Web Wallets and not App Wallets?  Uber and Lyft, for example, are now commonly referred to as "App-based Rides".